src/Access.hs

```diff
@@ -415,8 +415,8 @@ isSameSchoolClass ctx = isJust . isSameSchoolClass' (ctx ^. capCtxUser)
 isSameSchoolClass' :: User -> User -> Maybe SchoolClass
 isSameSchoolClass' user user' = if c == c' then c else Nothing
```

Comment:
this can be changed to the obvious `sharesSomeSchoolClasses`, but IRC at least one of the use cases is to filter ideas in the user profile for those the visitor has access two.  for that we'll need `isInAccessibleSchoolClass :: Idea -> User -> Maybe SchoolClass`.